### ISSUE TITLE
Create VZV summary statistics widgets to embed  in ATD website

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#7205]
The Transportation Department is launching an add campaign for Vision Zero in November and would like to include the dynamic summary statistics from the Vision Zero Viewer into their website.

In a meeting on 9/16 @johnclary, @mateoclarke, and I discussed with PIOs Tim Z., and Samantha S. on what it would take to embed the statistics in their Drupal site, what would align with responsive design, and how to link back to the VZV.

![Screen Shot 2021-09-17 at 12.11.45 PM.png](https://images.zenhubusercontent.com/5d68810b05d7792664be403f/9a94ea3a-b895-4cf3-b407-02a9c8db3fff)

We would like to create two versions of the widget. Create 5 new routes
- 1 - Add a new page with the “measure cards” that could be embedded as an iframe
- 4 - Create a page for each “measure card” that could embedded as individual iframes to allow for more flexibility

Once the widgets are ready we’ll need to coordinate with Tara O. (CPIO) or Jacob B. (PIO) to transfer them to the website, and make sure they link back to the VZV.

**Comment by patrickm02L** [comment]
We need to make sure who it's going to be handed off to? Jacob B? 

Would also like clarity on what the staging environment looks like, and who's testing.

We'll create a URL, and PIO would need to insert into the Drupal editor.

**Comment by patrickm02L** [comment]
Ping Samantha S. CPIO when the widgets are ready to go.

**Comment by patrickm02L** [comment]
@chiaberry and I chatted with Samantha to work through an issue with the right side of the widget boxes getting chopped off at smaller desktop view break points. 

We also discussed an alternative to why the individual widgets were rendering in a staggered, stacked way, which occurred because 2 i-frames were included on the same line in the CMS aligning one to the left and one to the right. Instead, Chia suggested creating a widget that had 2 measure cards per widget so that we could give them the 2-column look. 

This works great in the webpage view, but gets scrunched in the mobile view (see below). So, the two columns return back to one column in mobile. 

![Screen Shot 2021-10-15 at 2.15.01 PM.png](https://images.zenhubusercontent.com/5d68810b05d7792664be403f/ceab475f-4340-4e4b-92ce-b0eeba6f7e2a)

Sent the new URLs to Samantha to upload to the staging environment.

https://deploy-preview-1034--atd-vzv-staging.netlify.app/measures/fatalities-years

https://deploy-preview-1034--atd-vzv-staging.netlify.app/measures/injuries-totalcrashes
